packages/studio-ui/tests/components/UndoRedo.test.tsx

```diff
@@ -46,6 +56,26 @@ describe("Undo/redo", () => {
     expect(useStudioStore.getState().pages.activeComponentUUID).toBeUndefined();
   });
 
+  it("undoes last state update using control + z if OS is not OS X", async () => {
```

Comment:
can we also add (or update tests) tests to check that control-z won't work on OS and cmd-z won't work on Windows?

Reply:
done!